python/tvm/relay/frontend/tensorflow.py

```diff
@@ -626,8 +626,14 @@ def _impl(inputs, attr, params):
 
 def _slice():
     def _impl(inputs, attr, params):
-        begin = _get_list_param(params, inputs[1])
-        size = _get_list_param(params, inputs[2])
+        try:
+            begin = _get_list_param(params, inputs[1])
+        except:
```

Comment:
We should probably only catch the exceptions we expect to see here (like `IndexError` or `AttributeError`), and let the others go through.

Reply:
Got it